api/src/constants/email-templates/bceid/requestBceidCompleted/html.pug

```diff
@@ -0,0 +1,41 @@
+html
+  head
+    meta(name='viewport', content='width=device-width')
+    meta(http-equiv='Content-Type', content='text/html; charset=UTF-8')
+    style
+      include ../../email.css
+    title Realm-o-Matic Email Notification
+  body
+    table.body-wrap
+      tbody
+        tr
+          td.container
+            .content
+              table.main
+                tbody
+                  tr
+                    td.header BCeID Enabled for Production KeyCloak Realm
+                  tr
+                    td.content-wrap
+                      table.message-content
+                        tbody
+                          tr
+                            td Hi #{name},
+                          tr
+                            td As you have requested BCeID integration with Keycloak Realm #{realmName}, IDIM team has reviewed your application and approved on the usage.
```

Comment:
As you have requested BCeID integration with Keycloak Realm foo, **_the_** IDIM team has reviewed your application and approved on the usage.